build/debian-hyperkube-base/Makefile

```diff
@@ -58,7 +59,7 @@ push-manifest:
 
 cni-tars/$(CNI_TARBALL):
 	mkdir -p cni-tars/
-	cd cni-tars/ && curl -sSLO --retry 5 https://storage.googleapis.com/kubernetes-release/network-plugins/${CNI_TARBALL}
+	cd cni-tars/ && curl -sSLO --retry 5 https://github.com/containernetworking/plugins/releases/download/${CNI_VERSION}/${CNI_TARBALL}
```

Comment:
how about `https://storage.googleapis.com/k8s-artifacts-cni/` now that it exist?

Reply:
@thockin -- that's the plan, but we'll need to backfill all of the recent CNI plugin releases to the bucket first.